test/MLJ/unit.jl

```diff
@@ -92,9 +110,9 @@ end
         suggestions = Dict{Symbol}[]
         tuning, state = setup(;n_startup=3,space=space, suggest=suggestions)
 
-        output = MLJTuning.models!(tuning, testmodel, nothing, state, 0, 0)
+        output, _ = MLJTuning.models(tuning, testmodel, nothing, state, 0, 0)
 
-        @test output isa Vector{Tuple{Any, TreeParzen.Trials.Trial}}
+        @test output isa Vector{<:Tuple{Any, TreeParzen.Trials.Trial}}
```

Comment:
How come this was necessary?

Reply:
It was failing without the change. Julia's type system is not ["covariant"](https://docs.julialang.org/en/v1/manual/types/). So for example:

```julia
julia> [(4, 3), ] isa Vector{Tuple{Any, Any}}
false

julia> [(4, 3), ] isa Vector{<:Tuple{Any, Any}}
true
```
But I guess you probably knew that. I admit that I don't understand is why the old tests were passing.